docs/cip/README.md

```diff
@@ -1775,57 +1717,82 @@ for further discussion.
 
 <a name="feasible-parameters"></a>**Feasible protocol parameters**
 
-The table below documents a set of Leios protocol parameters that provided high
-throughput and reasonably fast settlement in the prototype Haskell and Rust
-simulations of Leios. The exact choice of parameters that would be adopted on
-the Cardano mainnet must be subject to discussion and consideration of
-tradeoffs.
+**Parameter Relationships and Network Assumptions**
+
+The key innovation in this specification is the relationship between the voting
+threshold ($\tau = 75\%$) and the diffusion period ($L_\text{diff}$). The high
```

Comment:
```suggestion
The key relation in the proposed protocol is between the voting
threshold ($\tau = 75\%$) and the propagation delay of EBs ($\Delta_\text{EB}$). The high
```

It's not only L_diff which is at play here and we must balance both L_vote and L_diff with assumed Delta_EB (and Delta_RB!).